dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -444,6 +444,7 @@ public enum ErrorCode {
 
   E7180(
       "The analytics outliers data does not exist. Please ensure analytics job was run and did not skip the outliers"),
+  E7181("Order by param `{0}` does not exist"),
```

Comment:
```suggestion
  E7181("Column `{0}` specified, in orderBy, does not exist"),
```

Reply:
not all columns are eligible for order by and still can exist. What about Column `{0}` specified, in orderBy, is not eligible for orderBy or does not exist

Reply:
Makes sense @d-bernat 👍